<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QWidget" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deposit Calc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 15pt &quot;Times New Roman&quot;;
background: rgb(0, 0, 0)</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="title">
    <string>Капитализация процентов</string>
   </property>
   <widget class="QRadioButton" name="radioButton_yes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="text">
     <string>Да</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_no">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
   color: white;
</string>
    </property>
    <property name="text">
     <string>Нет</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_calc_deposit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 3px solid gray rgb(255, 209, 0);
   color: white;
   border-radius: 30px;
   background-color: rgb(255, 150, 17);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="text">
    <string>р</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="percent_deposit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="inputMask">
    <string>99999</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="text">
    <string>Калькулятор доходности вкладов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time_deposit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="inputMask">
    <string>99999</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="text">
    <string>Срок вклада</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_deposit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="inputMask">
    <string>999999999999999999999</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="text">
    <string>месяцев</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="percent_deposit_nalog">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="inputMask">
    <string>99999</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="text">
    <string>Налоговая ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
   color: white;
</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
